Libraries/test/Amazon.Lambda.Annotations.SourceGenerators.Tests/SourceGeneratorTests.cs

```diff
@@ -83,6 +90,12 @@ public async Task SimpleCalculator()
                     }
                 }
             }.RunAsync();
+
+            var expectedTemplateContent = File.ReadAllText(Path.Combine("Snapshots", "ServerlessTemplates", "simpleCalculator.template"));
+            var actualTemplateContent = File.ReadAllText(Path.Combine("TestServerlessApp", "serverless.template"));
+            expectedTemplateContent = expectedTemplateContent.Replace("\r\n", "\n");
```

Comment:
```suggestion
            expectedTemplateContent = expectedTemplateContent.Replace("\r\n", System.Environment.NewLine);
            expectedTemplateContent = expectedTemplateContent.Replace("\n", System.Environment.NewLine);
```

this will ensure that expected template always have system based new line.

Reply:
You will ultimately end up with \r\r\n. I tried this and it didn't work.

Reply:
There needs be a better way, It seems solvable without modifying the actual template.

Reply:
Converting everything to unix based endings is the cleanest way (imo). Also, we are just comparing strings and not writing anything back to the template file.

Reply:
by converting the actual output to a different format you are changing the expectations (I didn't mean expected output here).

There should not a be a manipulation on the output to make the test pass.

Reply:
Added an extra replace on \r\r\n.